pkgs/by-name/dn/dness/package.nix

```
@@ -0,0 +1,46 @@
+{
+  fetchFromGitHub,
+  lib,
+  openssl,
+  pkg-config,
+  rustPlatform,
+  ...
+}:
+rustPlatform.buildRustPackage (
+  let
+    inherit (lib) licenses maintainers platforms;
+    name = "dness";
+    version = "v0.5.7";
+    src = fetchFromGitHub {
+      owner = "nickbabcock";
+      repo = "dness";
+      rev = version;
+      hash = "sha256-Vty4ec6aoUh3p2b9vLkNeS5R4pJWzjwYrC5DtVVyhT8=";
+    };
+  in
+  {
+    pname = name;
+    inherit src version;
+    nativeBuildInputs = [
+      pkg-config
+    ];
+    PKG_CONFIG_PATH = "${openssl.dev}/lib/pkgconfig";
+    cargoHash = "sha256-VyjntXsb1FUNguJFEmaZmjeCUHPMHMDICTTMDwExNBI=";
```

```suggestion

  cargoHash = "sha256-WhSeNukPjgM7Cy8LWi/s1YGa5/UxsFU1NGL7vIUlU58=";

```

Cargo hash changed for some reason.